src/plugins/gui/scenes/SepaTransferScene.tsx

```diff
@@ -0,0 +1,180 @@
+import Clipboard from '@react-native-clipboard/clipboard'
```

Comment:
This file has nothing to do with Sepa. Rename file as it is a generic information display scene. I'd choose

`InfoDisplayScene.tsx`